python_modules/dagster/dagster/core/storage/event_log/sqlite/sqlite_event_log.py

```diff
@@ -230,16 +230,13 @@ def store_event(self, event):
         with self.run_connection(run_id) as conn:
             conn.execute(insert_event_statement)
 
-        if (
-            event.is_dagster_event
-            and event.dagster_event.is_step_materialization
-            and event.dagster_event.asset_key
-        ):
+        if event.is_dagster_event and event.dagster_event.asset_key:
             # mirror the event in the cross-run index database
             with self.index_connection() as conn:
                 conn.execute(insert_event_statement)
 
-            self.store_asset(event)
+            if event.dagster_event.is_step_materialization:
+                self.store_asset(event)
```

Comment:
Add a comment here about how we're only registering materializations in the asset catalog?